content/blog/2018-04-22-essurvey.md

```diff
@@ -0,0 +1,337 @@
+---
+slug: "essurvey"
+title: Download data from the European Social Survey

```

Comment:
Wondering if you could modify the title since what you can do with the package is so much more interesting than "download". Maybe try to refer to types of questions you could address?